kext/FileNVRAM/FileNVRAM.h

```diff
@@ -180,7 +182,7 @@ class FileNVRAM : public IODTNVRAM
     OSDictionary * mNvramMissDict;
     IOCommandGate* mCommandGate;
     OSString*      mFilePath;
-    IOTimerEventSource* mTimer;
+    IONotifier* disk_notifier_start_;
```

Comment:
Ha ha, that was the problem now it works. So mutch trouble for a little typo :-) .